lib/network.sh

```diff
@@ -74,6 +74,7 @@ if [[ "$CLUSTER_APIENDPOINT_IP" == *":"* ]]; then
 else
   export CLUSTER_APIENDPOINT_HOST="$CLUSTER_APIENDPOINT_IP"
 fi
+export CLUSTER_APIENDPOINT_PORT=${API_ENDPOINT_PORT:-"6443"}
```

Comment:
I think the RHS variable `API_ENDPOINT_PORT` isn't set anywhere in the dev env. There's `CLUSTER_APIENDPOINT_IP` though, so I think it would make sense to match the naming on the LHS and RHS, call this `CLUSTER_APIENDPOINT_PORT` on both sides and document the name of this variable in vars.md.

Reply:
Done